src/CoreBundle/private/modules/MTTableEditor/views/includes/fields.inc.php

```diff
@@ -35,16 +34,18 @@
                     $oField->oDefinition->sqlData['modifier'] = 'readonly';
                 }
 
-                $sTmpFormTabsContent .= '<tr class="row m-0 '.$rowClass.'">
-            <th valign="top" class="leftTD col-2"'.$rowColorStyle.'>
-              <div id="fieldname_'.TGlobal::OutHTML($oField->name).'">
+
+                // todo $rowColorStyle
```

Comment:
Is this a TODO we can already work on? Ideally we don't have TODOs left in production code.

Reply:
oooh... missed that and found out i missed the handling of the separator field type as well. Now included.